packages/snaps-cli/src/config.ts

```diff
@@ -531,6 +531,45 @@ export const SnapsWebpackConfigStruct = object({
     {},
   ),
 
+  polyfills: defaulted(
```

Comment:
Since we don't infer the types from this config (because that makes it difficult to do docs), this needs to be added to the `WebpackConfiguration` type too.